app/models/user.rb

```diff
@@ -73,13 +73,16 @@ def is_data_protection_officer!
   RESET_PASSWORD_TEMPLATE_ID = "2c410c19-80a7-481c-a531-2bcb3264f8e6".freeze
   CONFIRMABLE_TEMPLATE_ID = "257460a6-6616-4640-a3f9-17c3d73d9e91".freeze
   BETA_ONBOARDING_TEMPLATE_ID = "b48bc2cd-5887-4611-8296-d0ab3ed0e7fd".freeze
+  USER_REACTIVATED_TEMPLATE_ID = "ac45a899-490e-4f59-ae8d-1256fc0001f9".freeze
 
   def reset_password_notify_template
     RESET_PASSWORD_TEMPLATE_ID
   end
 
   def confirmable_template
-    if was_migrated_from_softwire?
+    if last_sign_in_at.present? && (unconfirmed_email.blank? || unconfirmed_email == email)
+      USER_REACTIVATED_TEMPLATE_ID
+    elsif was_migrated_from_softwire?
       BETA_ONBOARDING_TEMPLATE_ID
```

Comment:
This looks odd, would we not want to send CONFIRMABLE_TEMPLATE_ID if the email address changes instead of BETA_ONBOARDING_TEMPLATE_ID after the user have used the system and decided to change the email?

Reply:
Separate ticket?